packages/python-aiohttp-xmlrpc/python-aiohttp-xmlrpc.spec

```diff
@@ -43,5 +43,8 @@ rm -rf %{pypi_name}.egg-info
 %{python3_sitelib}/aiohttp_xmlrpc-%{version}-py%{python3_version}.egg-info
 
 %changelog
+* Fri Jan 28 2022 Odilon Sousa <osousa@redhat.com> - 1.3.2-1
```

Comment:
this version already exists in 3.16, built against a different python, so you'd need to do `1.3.2-0.1`